app/lib/meadow_web/plugs/search.ex

```diff
@@ -68,10 +69,36 @@ defmodule MeadowWeb.Plugs.Search do
 
   defp extract_body(%Plug.Conn{body_params: %Plug.Conn.Unfetched{aspect: :body_params}} = conn) do
     {:ok, body, _conn} = read_body(conn)
-    body
+    fix_multiline_json(body, conn)
   end
 
   defp extract_body(conn) do
     conn.body_params
   end
+
+  defp fix_multiline_json(body, conn) do
+    if get_req_header(conn, "content-type") == ["application/x-ndjson"] do
+      body
+      |> String.trim()
+      |> String.split("\n")
+      |> Enum.map_join("\n", fn line ->
+        line
+        |> Jason.decode!()
+        |> add_track_total_hits()
+        |> Jason.encode!()
+      end)
+      |> Kernel.<>("\n")
+    else
+      body
```

Comment:
Don't we want to add `track_total_hits` to a single `_search` / `application/json` query as well? I can't remember if there's a reason we're limiting this to `_msearch`. (I get that we're going with what ReactiveSearch is actually doing, but I'm just asking for the sake of consistency.)

Seems like it would be easy enough to add
```suggestion
      body
      |> Jason.decode!()
      |> add_track_total_hits()
      |> Jason.encode!()
```
right here.

Reply:
Added